examples/minimum/main.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  name = "boldlink-minimum-r53-example.com"
+}
+
+module "minimum_route53" {
+  source         = "../../"
+  name           = local.name
+  create_records = true
```

Comment:
Hey  Patrick, I think we should remove this in the minimum since we are only creating a hosted zone